From 09a095ddefb4a8e7f4f8e15326f0c1954db2912e Mon Sep 17 00:00:00 2001
From: Yaz Saito <yasushi.saito@gmail.com>
Date: Tue, 3 Mar 2020 08:59:51 -0800
Subject: [PATCH] Fix wrong uses of python "is not"

    /opt/paraview/lib/python3.8/site-packages/_paraview.zip/paraview/web/protocols.py:2444: SyntaxWarning: "is not" with a literal. Did you mean "!="?
    /opt/paraview/lib/python3.8/site-packages/_paraview.zip/paraview/web/protocols.py:2489: SyntaxWarning: "is not" with a literal. Did you mean "!="?
---
 Web/Python/paraview/web/protocols.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Web/Python/paraview/web/protocols.py b/Web/Python/paraview/web/protocols.py
index 9f3b107d0cd..c7a8c8c7b20 100644
--- a/Web/Python/paraview/web/protocols.py
+++ b/Web/Python/paraview/web/protocols.py
@@ -2441,7 +2441,7 @@ def restructureProperties(self, groupList, propList, uiList):
             groupInfo = groupList[idx]
             group = groupInfo['group']
             parentGroup = groupInfo['parentGroup']
-            if group is not 'root' and self.groupDetailsMap[group]['groupVisibility'] == 'never':
+            if group != 'root' and self.groupDetailsMap[group]['groupVisibility'] == 'never':
                 self.debug('Culling property (%s) in group (%s) because group has visibility never' % (propList[idx]['name'], group))
                 continue
             if not group in groupMap:
@@ -2486,7 +2486,7 @@ def restructureProperties(self, groupList, propList, uiList):
         # case, the group should be marked the same.
         if uiList:
             for groupName in groupMap:
-                if groupName is not 'root' and 'ui' in groupMap[groupName]:
+                if groupName != 'root' and 'ui' in groupMap[groupName]:
                     groupUiList = groupMap[groupName]['ui']
                     firstUiElt = groupUiList[0]
                     groupDependency = False
